src/floating-focus.js

```diff
@@ -164,15 +170,41 @@ export default class FloatingFocus {
 		});
 	}
 
-	repositionElement(target, floater) {
+	standardizeFloat(number) {
+		return Math.round(parseFloat(number.toFixed(3) * 1000)) / 1000;
+	}
```

Comment:
[Same comment as https://github.com/Q42/floating-focus-a11y/pull/4/files#r353172959]

Reply:
Again I think it's fine to leave it here, but it might be too generic for this class. You decide 😄